File: frontend/src/app/model/prioritymodel.ts
export class PriorityModel {
  id: string;
  priority: string;

  static cloneBase(priority: PriorityModel): PriorityModel{
    const clonePriorityModel: PriorityModel = new PriorityModel();
    clonePriorityModel.id = priority.id;
    clonePriorityModel.priority = priority.priority;
    return clonePriorityModel;
  }
}

File: fapi/src/main/java/com/netcracker/edu/project/fapi/model/PriorityModel.java
package com.netcracker.edu.project.fapi.model;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

@JsonIgnoreProperties(ignoreUnknown = true)
public class PriorityModel {

    private int id;
    private String priority;

    public PriorityModel() {
    }

    public PriorityModel(int id, String priority) {
        this.id = id;
        this.priority = priority;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getPriority() {
        return priority;
    }

    public void setPriority(String priority) {
        this.priority = priority;
    }
}
